Gives service module some tweaks

mikomyazaki opened the pull request:
:cl: mikomyazaki
tweak: Adds basic medical and trained chemistry skills to the Service Module. Gives the Service Module the full selection of booze and soft drink dispenser drinks. Significantly reduces power requirement of the drink dispenser tool.
/:cl:

- Adds all drinks from the booze and soft drinks dispensers to their portable version. They were missing a few.
- Reduces power usage of their dispenser, so you don't burn 20-30% of the default battery making a single drink, which can happen if you use the 5u dispenser mode.
- Adds basic medical and trained chemistry skills, so the Service 'bot knows what they're serving is/isn't poison, and lets them use chem dispensers fully.

afterthought2 commented on code/modules/reagents/reagent_containers/borghydro.dm:
```diff
@@ -120,11 +120,11 @@
 	volume = 60
 	possible_transfer_amounts = "5;10;20;30"
-	reagent_ids = list(/datum/reagent/ethanol/beer, /datum/reagent/ethanol/coffee/kahlua, /datum/reagent/ethanol/whiskey, /datum/reagent/ethanol/wine, /datum/reagent/ethanol/vodka, /datum/reagent/ethanol/gin, /datum/reagent/ethanol/rum, /datum/reagent/ethanol/tequilla, /datum/reagent/ethanol/vermouth, /datum/reagent/ethanol/cognac, /datum/reagent/ethanol/ale, /datum/reagent/ethanol/mead, /datum/reag ...
+	reagent_ids = list(/obj/item/weapon/reagent_containers/chem_disp_cartridge/water,/obj/item/weapon/reagent_containers/chem_disp_cartridge/ice,/obj/item/weapon/reagent_containers/chem_disp_cartridge/coffee,/obj/item/weapon/reagent_containers/chem_disp_cartridge/hot_coco,/obj/item/weapon/reagent_containers/chem_disp_cartridge/cream,/obj/item/weapon/reagent_containers/chem_disp_cartridge/tea,/obj/it ...
```
Please format it so as to have one entry per line; hard to read like this.

mikomyazaki replied:
Done.

mikomyazaki committed:
Gives service module dispenser all the drinks from the booze and softdrinks machine.

Gives the service module some low medical and chemistry skills so they know what is and isn't fine to serve to the crew.

Reduces the amount of power need to pour a drink, should not take 20% power to pour a drink.

babydoll commented:
purpose of changelog is to describe changes in enough detail to be useful, not just to allude to them

mikomyazaki commented:
> purpose of changelog is to describe changes in enough detail to be useful, not just to allude to them

Thanks. Made it more descriptive.
